Deliverable 0 - Natural Disaster Prevention

icyair100 opened the pull request:
Deliverable  0 contains updated proposal.

icyair100 committed:
Deliverable 0 - Natural Disaster Prevention

Deliverable  0 contains updated proposal.

gallettilance reviewed (changes requested):
please nest your proposal within a folder named after your team name instead of having the file in the root of the repository

icyair100 committed:
Nest in Folder

Deliverable 0

icyair100 commented:
Changes Made

icyair100 committed:
weather data

icyair100 committed:
Merge pull request #1 from icyair100/AR_D1

weather data

Commit:
THis is a test, ok?

varocarras committed:
Update README.md

varocarras committed:
Update README.md

icyair100 committed:
HeatMap/Transform Data

Transform weather data into heat map and choropleth map

icyair100 committed:
Fixed Chloropleth Map

icyair100 committed:
HeatMap/Transform Data (#4)

* HeatMap/Transform Data

Transform weather data into heat map and choropleth map

* Fixed Chloropleth Map

varocarras committed:
Added simplified list of Tornado locations

Created by Abdullah

icyair100 committed:
Python script to get weather data from API

gathered data from API & stored in dataframe & csv

icyair100 committed:
python notebook fix

icyair100 committed:
Python script to get weather data from API (#6)

* HeatMap/Transform Data

Transform weather data into heat map and choropleth map

* Fixed Chloropleth Map

* Python script to get weather data from API

gathered data from API & stored in dataframe & csv

* python notebook fix

icyair100 committed:
Created README for Deliverable 1

Created Markdown version of Project proposal for easier future editing.

icyair100 committed:
Merge pull request #7 from icyair100/AR_D1

Created README for Deliverable 1

varocarras committed:
Merge branch 'master' into AC_D1

icyair100 committed:
Create .gitignore

icyair100 committed:
Merge pull request #8 from icyair100/AR_D1

Create .gitignore

varocarras committed:
Remove unnecessary files

Removing historical_data
Removing historical_data2
Removing weather_data

varocarras committed:
Merge pull request #9 from icyair100/AC_D1

Removing unnecessary files

fhorn97 committed:
Update Project_Proposal_CS506.md

fhorn97 committed:
Merge pull request #11 from icyair100/FH_D1

Updated Names in Project Proposal

icyair100 committed:
Deliverable 1

icyair100 committed:
Checklist update

icyair100 committed:
Merge pull request #15 from icyair100/Deliverable-1

Remove conflict files

icyair100 committed:
Delete .DS_Store

icyair100 committed:
Delete .DS_Store

icyair100 committed:
Delete .DS_Store

icyair100 committed:
Create README.md

icyair100 committed:
Merge branch 'Deliverable-1' of https://github.com/icyair100/CS506Spring2021Repository into Deliverable-1

icyair100 committed:
Merge pull request #16 from icyair100/Deliverable-1

Fix Master Branch

icyair100 commented:
Resubmitted on a new branch
